.travis.yml

```diff
@@ -9,18 +9,23 @@ notifications:
     secure: JKzk2sJSbZ9h2PUVWj6KtOAdFbEEnOtv/VZy05pJ2H41xRgUHiGdtMW/vMSeq6XX3IJN8eW2zd0cJTgkFn0ioAlYvID8zRhcvkFHg60QXquoqtp5y65dxjtVz79hefxSo7FO1NhMZBQWE9Tg6R7XkoyTMth62+T9vqOgu2Hms6M=
     if: (branch = main) AND (type = push)
     on_success: change # default: always
+before_script: pip install awscli --user
 jobs:
   include:
     - stage:
-      name: "Linting, Coverage, Deployment"
+      name: "Coverage"
+      script:
+        - npm --silent run build -- --node
```

Comment:
I don't think the `--node` target isn't necessary for coverage.

Reply:
Updated.